Karma-Khameleons/bangazon-website
# Sam order payment type

## Description
Users are now able to create payment types from the order detail view.


## Number of Fixes
Fixed link in success.html so that clicking "check out" takes the user to the order detail view.


## Related Ticket(s)
Fixed: #8 


## Problem to Solve
Customers should be able to select a payment type or create a new payment type from the order view

## Changes
Updated link in success.html and added an event listener to static/main.js
Removed superfluous "shopping cart" tab from the navbar


## Expected Behavior
Customers can create payment types from the order view


## Steps to Test Solution
Login as a registered user, add items to your order, and checkout. New payment types created from the order detail view should be useable on the order.

## Unit Testing

## Documentation


## Deploy Notes
No migrations should be needed
